Prova correzione errore di invio

Alessio-Gulmini-JCMaxwell-5BI opened the pull request:
Ho provato a correggere l'errore presente nella richiesta.

Alessio-Gulmini-JCMaxwell-5BI committed:
Prova correzione errore di invio

Ho provato a correggere l'errore presente nella richiesta.

Alessio-Gulmini-JCMaxwell-5BI commented:
@Luca-Volturno-JCMaxwell-5BI

Luca-Volturno-JCMaxwell-5BI commented:
Arrivo a casa e la provo, non credo però che sia questa la soluzione al problema, ti farò sapere

Alessio-Gulmini-JCMaxwell-5BI committed:
Prova correzione errore 2.0

Description in title

Alessio-Gulmini-JCMaxwell-5BI commented:
@Luca-Volturno-JCMaxwell-5BI Guarda tra i commits, ho provato a ricorreggere l'errore.

Alessio-Gulmini-JCMaxwell-5BI commented:
Rip

Alessio-Gulmini-JCMaxwell-5BI commented:
@Luca-Volturno-JCMaxwell-5BI Non ho ben capito come fare per correggere l'errore, se ne fossi in grado riusciresti a darmi una mano?

Luca-Volturno-JCMaxwell-5BI committed:
Merge branch 'Luca-Volturno-JCMaxwell-5BI-patch-Nickname-v1.0' into Alessio-Gulmini-JCMaxwell-5BI-patch-Nickname-v.1.0

Luca-Volturno-JCMaxwell-5BI commented:
> @Luca-Volturno-JCMaxwell-5BI Non ho ben capito come fare per correggere l'errore, se ne fossi in grado riusciresti a darmi una mano?

Tip: Prova a smanettare sul codice, butta qualche stringa fuori o dentro alla if

Alessio-Gulmini-JCMaxwell-5BI committed:
Correzione Errore Invio

Ho corretto l'errore inerente all'invio presente prima della richiesta del Nickname.

Luca-Volturno-JCMaxwell-5BI commented:
@Alessio-Gulmini-JCMaxwell-5BI  Ottimo hai trovato la soluzione procediamo con il merge nell'altro branch
